macos/file_picker.podspec

```diff
@@ -0,0 +1,25 @@
+#
+# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html.
+# Run `pod lib lint file_picker.podspec` to validate before publishing.
+#
+Pod::Spec.new do |s|
+  s.name             = 'file_picker'
+  s.version          = '0.0.1'
+  s.summary          = 'A flutter plugin to show native file picker dialogs'
+  s.description      = <<-DESC
+A flutter plugin to show native file picker dialogs
+                       DESC
+  s.homepage         = 'https://github.com/miguelpruivo/plugins_flutter_file_picker'
+  s.license          = { :file => '../LICENSE' }
+  s.author           = 'Miguel Ruivo, Dominik Roszkowski'
+  s.source           = { :path => '.' }
+  s.source_files = 'Classes/**/*'
+
+  s.resource_bundles = {'file_picker_privacy' => ['Resources/PrivacyInfo.xcprivacy']}
+
+  s.dependency 'FlutterMacOS'
+
+  s.platform = :osx, '10.11'
```

Comment:
Just to double-check but does this align with Flutter's minimum for MacOS for the current minimum Flutter version that we support?

Reply:
Created with `flutter create -t plugin` on current stable, so assumed it should follow the current requirements. Does that sound right?

Reply:
I think this should be fine. We support Flutter 3.22 or higher, and that Flutter version requires a higher MacOS minimum, so it's Flutter that will dictate the minimum MacOS version.